packages/react-i18n/docs/react_i18n_schema.md

```diff
@@ -22,6 +27,14 @@ React I18n is a schema comprised of a structure of nested key-value pairs (KVPs)
 
 # History
 
+## Version 2.0
```

Comment:
It is unclear to me if you simply didn't mention this support in all of our current parsers or if there are any parsers actually implemented with the 1.0 spec that _aren't_ compatible with this new assumption? We only need to define a new spec if anyone is actually affected by this amendment.

Reply:
🤷 You're right. There probably aren't any.

However, I like to treat specification documents as "append only". This could have been considered errata, or perhaps the `1.0` document should have been in some form of "draft" mode until we had confidence that all the details had been ironed out. 

In the end, versions numbers are cheap. 🤷

Reply:
A major version jump seems excessive to me when there are no breaking changes. I would rather use `1.1`, but I'll leave that up to you.